scaffolder-templates/basic-workflow/skeleton/src/main/resources/assessment-template.sw.yaml

```diff
@@ -8,6 +8,33 @@ annotations:
 dataInputSchema:
   schema: schemas/${{ values.artifactId }}-input-schema.json
   failOnValidationErrors: true
+extensions:
+  - extensionid: workflow-output-schema
+    outputSchema:
+      schema: schemas/${{ values.artifactId }}-workflow-result-schema.json
```

Comment:
Is there a place in the template's description we can specify the orchestrator plugin version that supports this schema?

Reply:
In the [README](https://github.com/parodos-dev/workflow-software-templates/blob/main/README.md) maybe?

Reply:
makes sense. a compatibility section will be useful there